lib/public/views/Environments/format/displayEnvironmentStatusHistory.js

```diff
@@ -0,0 +1,68 @@
+/**
+ *  @license
+ *  Copyright CERN and copyright holders of ALICE O2. This software is
+ *  distributed under the terms of the GNU General Public License v3 (GPL
+ *  Version 3), copied verbatim in the file "COPYING".
+ *
+ *  See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ *  In applying this license CERN does not waive the privileges and immunities
+ *  granted to it by virtue of its status as an Intergovernmental Organization
+ *  or submit itself to any jurisdiction.
+ */
+import { h } from '/js/src/index.js';
+
+/**
+ * Display the given environment's status history string
+ *
+ * @param {Environment} environment the environment for which status should be displayed
+ * @return {vnode} the resulting component
+ */
+export const displayEnvironmentStatusHistory = (environment) => {
+    const { historyItems } = environment;
+
+    const statusAcronyms = {
+        STANDBY: 'S',
+        DEPLOYED: 'DL',
+        CONFIGURED: 'C',
+        RUNNING: 'R',
+        ERROR: 'E',
+        MIX: 'M',
+        DESTROYED: 'DS',
+    };
+
+    /**
+     * Get correct color class from environmentname.
+     * @param {*} status One of the Environment statuses
+     * @returns {string} The classname used to color the acronoym.
+     */
+    const getColorClassEnvironment = (status) => {
+        switch (status) {
+            case 'RUNNING':
+                return '.success';
+            case 'ERROR':
+                return '.danger';
+            case 'CONFIGURED':
+                return '.warning';
+            default:
+                return '';
+        }
+    };
+
+    const statusHistory = [];
+
+    // Iterate through historyItems
+    for (const { status } of historyItems) {
+        if (status in statusAcronyms) {
+            statusHistory.push({ status, content: statusAcronyms[status] });
+            statusHistory.push({ status: null, content: '-' });
```

Comment:
If the purpose is simply to add a separator between the statuses, you should look at adding it in the display component below as `null` status is not a valid one.

Reply:
- Added seperator in display component, used ternary operator to determine whether seperator should be shown. To avoid trailing seperator.